modules/experimental/eirini_release.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
-set -ex 
 
-if [ "$KIND_VERSION" != "v0.4.0" ]; then
- err "We only support Kind version 0.4.0 for now"
-
- exit 1
+. ./defaults.sh
+. ../../include/common.sh
+. .envrc
 
+if [ -f "$BUILD_DIR"/bin/kind ] && [ "$("BUILD_DIR"/bin/kind version)" != "v0.4.0" ]; then
```

Comment:
@mudler is this still needed?

Reply:
the only version tested there was `v0.4.0`

Reply:
but at the times, it was to check that you were running the latest (and greatest)

Reply:
So it could be removed actually